src/ensembl/src/content/app/species-selector/hooks/useSpeciesSelectorAnalytics.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useSelector } from 'react-redux';
+
+import analyticsTracking from 'src/services/analytics-service';
+
+import { getSpeciesAnalyticsName } from 'ensemblRoot/src/content/app/species-selector/speciesSelectorHelper';
+
+import { getCommittedSpecies } from 'ensemblRoot/src/content/app/species-selector/state/speciesSelectorSelectors';
+
+import {
+  PopularSpecies,
+  SearchMatch
+} from 'src/content/app/species-selector/types/species-search';
+import { CurrentItem } from 'src/content/app/species-selector/state/speciesSelectorState';
+
+enum AnalyticsCategories {
+  SELECTED_SPECIES = 'selected_species',
+  SPECIES_SELECTOR = 'species_selector',
+  POPULAR_SPECIES = 'popular_species',
+  SPECIES_SEARCH = 'species_search'
+}
```

Comment:
What's the purpose of this enum? We are just using it once in this file, aren't we? And not for any type checking purposes either? Why wouldn't we use just strings, like what you use for the `action` field?

_(same for the species page file)_

Reply:
That's done.